src/api/entrypoints/new_password/views.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, Depends, status
+from src.api.database.session import get_db
+from src.api.entrypoints.new_password.schema import NewPasswordChange, NewPasswordCodeAuth, NewPasswordRequest
+from src.api.services.new_password import ServiceNewPassword
+from sqlalchemy.orm import Session
+
+router = APIRouter()
+
+
+@router.post("/", response_model=NewPasswordChange, status_code=status.HTTP_200_OK)
+def set_new_password(request: NewPasswordChange, db: Session = Depends(get_db)):
```

Comment:
```suggestion
async def set_new_password(request: NewPasswordChange, db: Session = Depends(get_db)):
```
Acredito que todos endpoints deveriam ser assíncronos.